staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -1176,6 +1176,11 @@ type Interface struct {
 	// If specified, the virtual network interface address and its tag will be provided to the guest via config drive
 	// +optional
 	Tag string `json:"tag,omitempty"`
+	// If specified, the ACPI index is used to provide network interface device naming, that is stable across changes
+	// in PCI addresses assigned to the device.
+	// This value is required to be unique across all devices and be between 1 and (16*1024-1).
+	// +optional
+	ACPIIndex int `json:"acpiIndex,omitempty"`
```

Comment:
int32 maybe ? as in api/openapi-spec/swagger.json

Reply:
JSON itself does not specify such types [1], I think this is a specific implementation of swagger.

I do not think we follow any rules here, as I saw things used at this level as signed, unsigned and with bit count specific.
I do not think this is something really important, having the simplest one should be fine.

[1] https://json-schema.org/understanding-json-schema/reference/numeric.html#integer